### DISCUSSION TITLE
Adding more sections

### DISCUSSION THREAD

**Discussion by haotian2006** [haotian2006/MyDocs#22] [Ideas]
Instead of just tags we could add more resources like my original docs and maybe even add a library section where we can showcase other people's libraries and provide documentation for it

**Comment by cyrus01337** [comment]
Your docs should be merged with the rest of the project, are they not displaying?

**Reply by haotian2006** [reply]
They are displaying I mean like adding a specific section for them

**Reply by cyrus01337** [reply]
Ah that makes more sense, yeah absolutely. Eventually there will be a side menu that displays all docs associated with a specific section.
